qiskit_optimization/algorithms/multistart_optimizer.py

```diff
@@ -52,7 +52,8 @@ def __init__(self, trials: int = 1, clip: float = 100.) -> None:
                 with the ``-clip`` or ``clip`` values correspondingly for the initial guesses.
         """
         super().__init__()
-        self._trials = trials
+        
+        self._trials = trials if trials > 0 else 1
```

Comment:
If trials <= 0 is really an error in regards of not being a valid value should we rather raise a ValueError, rather than 'hiding' the fact that this is being configured incorrectly i.e. by forcing it to 1 in the constructor or ignoring it as in the setter.

Reply:
We can do it both ways. Tell me if you want to raise error. Can do that too!

Reply:
@adekusar-drl Any thought on the above?

Reply:
Since the default value of trials is 1 in the constructor so, when the user gives non-positive value if we assign 1 to self._trials would not be something completely unexpected. On the other hand, your point of 'hiding' the fact is also valid. Just tell me which approach you want to go with and I will do it!

Reply:
I'd agree, raising an exception is better.